scripts.js

```diff
@@ -56,54 +53,74 @@ const changeMarker = () => {
   for (i=0; i < squares.length; i++) {
 
-    // will log out the id of each square as it loops over them.
     console.log(squares[i].id)
 
-    // sets the innerHTML to null to replace the "X" or "O"
     squares[i].innerHTML = null
+
   }  
+  currentMarker = 'X';
 }
 
-function checkWin(){
-  topLeft = document.getElementById('top-left').innerHTML;
-  topMid = document.getElementById('top-middle').innerHTML;
-  topRight = document.getElementById('top-right').innerHTML;
-  midLeft = document.getElementById('middle-left').innerHTML;
-  midMid = document.getElementById('middle-middle').innerHTML;
-  midRight = document.getElementById('middle-right').innerHTML;
-  topLeft = document.getElementById('bottom-left').innerHTML;
-  topMid = document.getElementById('bottom-middle').innerHTML;
-  topRight = document.getElementById('bottom-right').innerHTML;
-
-  if(topLeft == currentMarker && topMid == currentMarker && topRight == currentMarker){
-    alert(currentMarker + " wins!");
-  }
-  else if(midLeft == currentMarker && midMid == currentMarker && midRight == currentMarker){
-    alert(currentMarker + " wins!");
-  }
-  else if(botLeft == currentMarker && botMid == currentMarker && botRight == currentMarker){
-    alert(currentMarker + " wins!");
-  }
-  else if(topLeft == currentMarker && midLeft == currentMarker && botLeft == currentMarker){
-    alert(currentMarker + " wins!");
-  }
-  else if(topMid == currentMarker && midMid == currentMarker && botMid == currentMarker){
-    alert(currentMarker + " wins!");
+
+const checkForWin = () => {
+  if(horizontalWin() || verticalWin() || diagonalWin()) {
+    window.alert(`Player ${currentMarker} won!`)
+  } else {
+    changeMarker()
   }
-  else if(topRight == currentMarker && midRight == currentMarker && botRight == currentMarker){
-    alert(currentMarker + " wins!");
+}
+
+
+
+
+const horizontalWin = () => {
+  if((board[0][0] == "X" && board[0][1] == "X" && board[0][2] == "X") 
+  || (board[0][0] == "O" && board[0][1] == "O" && board[0][2] == "O")){
+   alert(currentMarker + " wins");
   }
-  else if(topLeft == currentMarker && midMid == currentMarker && botRight == currentMarker){
-    alert(currentMarker + " wins!");
+  else if((board[1][0] == "X" && board[1][1] == "X" && board[1][2] == "X") 
+  || (board[1][0] == "O" && board[1][1] == "O" && board[1][2] == "O")){
+    alert(currentMarker + " wins");}
+  else if((board[2][0] == "X" && board[2][1] == "X" && board[2][2] == "X") 
+  || (board[2][0] == "O" && board[2][1] == "O" && board[2][2] == "O")){
+    alert(currentMarker + " wins");
+
   }
-  else if(topRight == currentMarker && midMid == currentMarker && botLeft == currentMarker){
-    alert(currentMarker + " wins!");
+  
+};
+
+const verticalWin = () => {
+  if((board[0][0] == "X" && board[1][0] == "X" && board[2][0] == "X") 
+  || (board[0][0] == "O" && board[1][0] == "O" && board[2][0] == "O"))
+  alert(currentMarker + " wins");
```

Comment:
this works but you should include the curly brackets` { }`
 `if((board[0][0] == "X" &amp;&amp; board[1][0] == "X" &amp;&amp; board[2][0] == "X")   || (board[0][0] == "O" &amp;&amp; board[1][0] == "O" &amp;&amp; board[2][0] == "O")){
alert(currentMarker + " wins");
}`